src/content/blog/web-framework-performance-report.mdx

```diff
@@ -0,0 +1,127 @@
+
+LCP is the hardest of the three metrics to master. Only 52% of all websites tested pass this metric. Of our six frameworks tested, only Astro and SvelteKit beat this average. The rest come in below the average.
+
+# Coming Soon: Interaction to Next Paint (INP)
+
+**Interaction to Next Paint (INP)** is an experimental web vital that assesses responsiveness. INP observes the latency of all interactions a user has made with the page, and reports a single value which all (or nearly all) interactions were below. A low INP means the page was consistently able to respond quickly to all—or the vast majority—of user interactions.
+
+While INP is not an official core web vital today, the Chrome team has signaled [their hope](https://www.youtube.com/watch?v=Mizzbsvv8Os&t=378s) to replace First Input Delay (FID) with INP as the best measurement of responsiveness. If this happens, INP will become the third official core web vital.
+
+So, how do our frameworks stack up against this new metric?
+
+<CWVINPChart />
+
+Most noticible in the chart is that a good INP measurement is overall _much_ harder for our frameworks to achieve than First Input Delay (FID). While every framework tested saw an 80%+ pass rate of FID, no framework was able to see that same 80% pass rate on INP. Astro came closest, at 68.8% passing.
+
+Why is this? To quote Anne Burnes of RebelMouse:
+
+> FID quantifies a user's experience when trying to interact with unresponsive pages, but it only measures the first interaction. According to Google, INP takes a more well-rounded measurement of a site’s responsiveness by covering a site’s entire spectrum of interactions, from the time a page first begins to load until the user leaves a page. This comprehensive measurement makes INP a more reliable indicator of a site’s overall responsiveness than FID.
+>
+> The holistic nature of INP makes it more challenging to solve than FID, because your code has to be implemented in a way that protects responsiveness for the user during their entire journey, not just on first load. Since many interactions are done through JavaScript, it means your site has to be loaded carefully for optimized performance.
+>
+> -- Anne Burnes, [RebelMouse](https://rebelmouse.com/)
+
+While all six of our frameworks struggle with this metric, it's worth noting that the average pass rate across all tracked websites is a suprisingly high 60.9%. While Astro and Wordpress look like the standout successes in the chart above, these sites are performing only slightly above the industry average.
+
+This will be an interesting metric to watch in 2023, and Google continues to weigh adding FID as an official Core Web Vital.
```

Comment:
I know it’s a bit early, so not doing a full review, but flagging this typo:
```suggestion
This will be an interesting metric to watch in 2023, and Google continues to weigh adding INP as an official Core Web Vital.
```